about.html

```diff
@@ -390,11 +390,12 @@
     twitter_handle: JohanMabille
     gh_handle: JohanMabille
     subproject: Jupyter Kernels
-  - name: Isabela Presedo-Floyd
-    photo: Isabela_Presedo-Floyd.jpg
```

Comment:
Note I did not delete the photo from the repo because it is used in other places